packages/core/http/core-http-resources-server-mocks/src/http_resources_server.mock.ts

```diff
@@ -7,7 +7,25 @@
  */
 
 import { httpServerMock } from '@kbn/core-http-server-mocks';
-import { HttpResources, HttpResourcesServiceToolkit } from './types';
+import type { PublicMethodsOf } from '@kbn/utility-types';
+import { HttpResourcesService } from '@kbn/core-http-resources-server-internal';
+import type { HttpResources, HttpResourcesServiceToolkit } from '@kbn/core-http-resources-server';
+
+export type HttpResourcesMock = jest.Mocked<PublicMethodsOf<HttpResourcesService>>;
+
+function createHttpResourcesService() {
```

Comment:
I added the mock to be consistent with other core services mocks. The mock isn't used anywhere in existing code so I'm fine with removing it if needed.

Reply:
Fine keeping it for consistency with our other mocks ihmo